cmd/pager.go

```diff
@@ -0,0 +1,13 @@
+package cmd
```

Comment:
Do we need this file?

Reply:
nooope that was me moving too fast/added it by mistake